docs/tracing.rst

```diff
@@ -0,0 +1,144 @@
+.. _tracing:
+
+Tracing
+=======
+
+.. contents::
+    :depth: 3
+
+
+Overview
+--------
+
+It provides an instrument tracing wrapper function to use to instrument functions manually in pubtools-* projects.
+
+
+Usage
+.....................
```

Comment:
I believe the number of characters in these header lines need to match exactly the line above in order for docs to render properly and without warnings.

So, for example, since "Usage" is 5 characters long this needs to be:

```suggestion
.....
```

Reply:
Fixed it.